docs/corsair-commander-core-guide.md

```diff
@@ -10,14 +10,21 @@ The device should be initialized every time it is powered on.
 ```
 # liquidctl initialize
 Corsair Commander Core (experimental)
-├── Firmware version            1.6.135  
+├── Firmware version            2.6.201  
 ├── AIO LED count                    29  
 ├── RGB port 1 LED count              8  
 ├── RGB port 2 LED count              8  
 ├── RGB port 3 LED count            N/A  
 ├── RGB port 4 LED count            N/A  
 ├── RGB port 5 LED count            N/A  
 ├── RGB port 6 LED count            N/A  
+├── AIO Pump                        Yes  
```

Comment:
Should AIO pump become just AIO?

Reply:
I think that makes sense.  Or maybe even `AIO port`.

Also, since these are all booleans, maybe `AIO port connected` or `AIO port in use`?  (And similarly for the fan ports).